lib/response-interceptor-wrapper.js

```diff
@@ -35,7 +36,7 @@ function responseInterceptorResolve (response, instance) {
       local.backupOptions.method || config.method;
 
     config.baseURL = undefined;
-    config.url = response.headers['location'];
+    config.url = response.headers['location'] && url.resolve(response.config.url, response.headers['location']);
```

Comment:
I like `config.url = url.resolve(response.config.url, response.headers['location'] || '');`
Please change it?